crates/wit-parser/src/decoding.rs

```diff
@@ -359,16 +359,16 @@ impl DecodedWasm {
     /// Returns the [`Resolve`] for WIT types contained.
     pub fn resolve(&self) -> &Resolve {
         match self {
-            DecodedWasm::WitPackage(resolve, _) => resolve,
+            DecodedWasm::WitPackages(resolve, _) => resolve,
             DecodedWasm::Component(resolve, _) => resolve,
         }
     }
 
-    /// Returns the main package of what was decoded.
-    pub fn package(&self) -> PackageId {
+    /// Returns the main packages of what was decoded.
+    pub fn packages(&self) -> Vec<PackageId> {
```

Comment:
This might be best to return `&[PackageId]` instead of a `Vec`. You can turn `&T` into `&[T]` with [`slice::from_ref`](https://doc.rust-lang.org/stable/std/slice/fn.from_ref.html)

Reply:
Done.